CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 * BraintreeShopperInsights (BETA)
   * Add `shopperSessionID` to `BTShopperInsightsClient` initializer
   * Add `isPayPalAppInstalled()` and/or `isVenmoAppInstalled()`
+  * Replace `sendPayPalPresentedEvent()` and `sendPayPalPresentedEvent()` with `sendPresentedEvent(for:presentmentDetails:)`
```

Comment:
This entire shopper insights section should be moved to `## unreleased`, it's currently under `6.25.0` and was not released in that version.